winmake.ps1

```diff
@@ -0,0 +1,86 @@
+# Targets
+function Podman-Remote{
+    New-Item  ./bin/windows -ItemType Directory -ea 0
+
+    $buildInfo = Get-Date -UFormat %s -Millisecond 0
+    $buildInfo = "-X github.com/containers/podman/v4/libpod/define.buildInfo=$buildInfo "
+    $commit = Git-Commit
+    $commit = "-X github.com/containers/podman/v4/libpod/define.gitCommit=$commit "
+
+    Write-Host go build --ldflags "$commit $buildInfo " --tags "$remotetags" --o .\bin\windows\podman.exe ./cmd/podman/.
+    go build --ldflags "$commit $buildInfo " --tags "$remotetags" --o .\bin\windows\podman.exe ./cmd/podman/.
+}
+
+function Make-Clean{
+     Remove-Item ./bin -Recurse -Force -Confirm:$false
+}
+
+function Local-Machine {
+    param (
+    [string]$files
+    );
+    Build-Ginkgo
+    if ($files) {
+         $files = " --focus-file $files "
+    }
+
+    ./test/tools/build/ginkgo.exe -vv  --tags "$remotetags" -timeout=90m --trace --no-color $files pkg/machine/e2e/.
```

Comment:
Needs an exit code check here ($LASTEXITCODE) (note that $ErrorActionPreference = 'Stop' won't auto-trigger an abort on a command exec failure). Using a wrapper like in the above wrapper func comment can make this easier. Another way is to have an exit code check function you just call after every direct call